# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1382877245 14400
#      Sun Oct 27 08:34:05 2013 -0400
# Node ID 7900595f1533adc9881a77ed1b565b7bb881e86d
# Parent  7690c3477da1bd492a4970bca729df8a3d736ee4
* install.txi: Use @ref instead of xref or pxref in mid sentence.

diff --git a/doc/interpreter/install.txi b/doc/interpreter/install.txi
--- a/doc/interpreter/install.txi
+++ b/doc/interpreter/install.txi
@@ -510,24 +510,24 @@ command
 make CFLAGS=-O CXXFLAGS=-O LDFLAGS=
 @end example
 
 @noindent
 instead of just @command{make}.
 
 @item
 If you encounter errors while compiling Octave, first
-@xref{Installation Problems} for a list of
+see @ref{Installation Problems} for a list of
 known problems and if there is a workaround or solution for
-your problem.  If not,
+your problem.  If not, see
 @ifclear INSTALLONLY
-@pxref{Trouble},
+@ref{Trouble}
 @end ifclear
 @ifset INSTALLONLY
-see the file BUGS
+the file BUGS
 @end ifset
 for information about how to report bugs.
 
 @item
 Once you have successfully compiled Octave, run @code{make install}.
 
 This will install a copy of Octave, its libraries, and its documentation
 in the destination directory.  As distributed, Octave is installed in
